core/web/loader/chain.go

```diff
@@ -15,6 +16,7 @@ type chainBatcher struct {
 	app chainlink.Application
 }
 
+// DEPRECATED: loadByChainIDs is deprecated and we should be using loadByRelayIDs.
```

Comment:
nit/ no need for caps here: https://go.dev/wiki/Deprecated
```suggestion
// Deprecated: use loadByRelayIDs
```

Reply:
Done!